crates/electrum/tests/test_electrum.rs

```diff
@@ -22,10 +23,34 @@ fn get_balance(
     Ok(balance)
 }
 
+fn electrum_sync<I>(
+    client: &BdkElectrumClient<Client>,
+    spks: &[ScriptBuf],
+    batch_size: usize,
+    chain: &mut LocalChain,
+    graph: &mut IndexedTxGraph<ConfirmationBlockTime, I>,
+) -> anyhow::Result<SyncResult>
+where
+    I: Indexer,
+    I::ChangeSet: Default + Merge,
```

Comment:
**Nit:** This removes the unnecessary reference and allows more flexibility for what we can input in `spks`.

```suggestion
fn electrum_sync<I, Spks>(
    client: &BdkElectrumClient<Client>,
    spks: Spks,
    batch_size: usize,
    chain: &mut LocalChain,
    graph: &mut IndexedTxGraph<ConfirmationBlockTime, I>,
) -> anyhow::Result<SyncResult>
where
    I: Indexer,
    I::ChangeSet: Default + Merge,
    Spks: IntoIterator<Item = ScriptBuf>,
    Spks::IntoIter: ExactSizeIterator + Send + 'static,

```